### PR TITLE
R4R: optimize switch client strategy

### PR DESCRIPTION
**Pull request by HaoyangLiu**

### CODE REVIEW

File: relayer/daemon.go [hunk 1]
```diff
@@ -13,9 +14,10 @@ import (
 )
 
 const (
-	IgnoredTimeGap       = 1800
-	BatchSize            = 100
-	WaitSecondForTrackTx = 10
+	IgnoredTimeGap         = 1800
+	BatchSize              = 100
+	WaitSecondForTrackTx   = 10
+	UnconfirmedTxThreshold = 50
```
Comment (yutianwu) [inline relayer/daemon.go:20]: can we make it configurable so if we want to change that, we don't need to change code

Comment (HaoyangLiu) [reply relayer/daemon.go:20]: According to our experience, we might not need to change its value. Besides, current implementation doesn't require us to update congfiguration before update binary.

**Comment by guagualvcha** [comment]
can we only alert only need manually check?

**Comment by HaoyangLiu** [comment]
I removed auto claim tg message and I will change the undelivered package theashold from 10 to 100. Thus there will be no false alarm.

**Review by guagualvcha — APPROVED** [review]

**Review by yutianwu — APPROVED** [review]

**Pull request merged**